### PR TITLE
40/buttons

### PR DESCRIPTION
**Pull request by dustinleblanc**
You can review them in the styleguide locally, however the styleguide won't build on the PR environment

### CODE REVIEW

File: assets/scss/components/_button.scss [hunk 1]
```diff
@@ -1,5 +1,26 @@
 .btn {
-  border-radius: 4px;
+  border-radius: 0;
```
Comment (dustinleblanc) [inline assets/scss/components/_button.scss]: Might be redundant now...

File: assets/scss/components/_button.scss [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
   font-weight: 500;
-  line-height: inherit;
 }
+
+.btn-sm {
+  line-height: 1;
```
Comment (dustinleblanc) [inline assets/scss/components/_button.scss]: Maybe I should extract this to `.btn`...

File: assets/css/fonts.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+@font-face {
+  font-family: 'Poets Electra Roman';
+  src: url('../fonts/PoetsElectra-Italic.woff2') format('woff2'),
+  url('../fonts/PoetsElectra-Italic.woff') format('woff');
+  font-weight: normal;
+  font-style: italic;
+}
+
+
```
Comment (serundeputy) [inline assets/css/fonts.css]: is it intentional to add this css file and remove the scss version?

Comment (dustinleblanc) [reply assets/css/fonts.css]: Yep, it makes the storybook stuff buildable to separate it out

**Pull request merged**